exodus/analysis_query/models.py

```diff
@@ -1,9 +1,10 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
-
+f
```

Comment:
What is this `f?

Reply:
It is a typo.
Let me just remove it and modify the corresponding commit.